Library/Homebrew/cmd/audit.rb

```diff
@@ -855,6 +855,14 @@ def audit_line(line, lineno)
       if line =~ /(require ["']formula["'])/
         problem "`#{$1}` is now unnecessary"
       end
+
+      if line =~ /#\{share\}\/#{formula.name}/
+        problem "Use \#{pkgshare}\ instead of \#{share}\/#{formula.name}"
+      end
+
+      if line =~ /share\/"#{formula.name}"/
+        problem "Use pkgshare instead of (share/\"#{formula.name}\")"
```

Comment:
Unless anyone really strongly objects here I'm going to leave it as-is. If it turns out we get a bunch of people going "Huh, but that's not the usage I'm looking for" I'll go in and consider something else, but I do think the chances are low. (Famous last words, I'm sure).